README.md

```diff
@@ -27,11 +25,46 @@ Simulation environment using O3DE.
 
 ## Starting guide
 
-### Real robot configuration
+### Introduction
+
+The demo is provided as a binary package for Ubuntu 24.04 with ROS 2 Jazzy, ensuring quick and easy setup. The binary package is the recommended method to run the demo. The source code and a Dockerfile are also available for those who prefer to build the project themselves or run it in a containerized environment.
+
+### Building using docker environment
+
+The _Dockerfile_ defined in [docker folder](./docker) will prepare the appropriate ROS 2 distribution based environment and build the components necessary to run the demo project simulator through the O3DE engine.
+
+#### Requirements
+
+* [Hardware requirements of o3de](https://www.o3de.org/docs/welcome-guide/requirements/)
+* Ubuntu 22.04 (Jammy) or newer
+* At least 60 GB of free disk space
+* Docker installed and configured
+  * **Note** It is recommended to have Docker installed correctly and in a secure manner so that the docker commands in this guide do not require elevated privileges (sudo) in order to run them. See [Docker Engine post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) for more details.
+* [Nvidia container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+#### Building the Full Docker Image
+
+The `Dockerfile` supports defining which version of Ubuntu+ROS to base the docker container on, and by default will support Ubuntu 24.04 (noble) with the ROS 2 Jazzy distribution. It will build a Docker image that will contain the Editor and the demo launcher. Use the following command to build the image:
+
+```
+docker build -t rai-rosbot -f Dockerfile .
```

Comment:
Dockerfile is located in the `docker` directory

Reply:
Added, thank you.